installing/installing_cloud_other/install-secured-cluster-cloud-other.adoc

```diff
@@ -0,0 +1,95 @@
+:_content-type: ASSEMBLY
+[id="install-secured-cluster-cloud-other"]
+= Installing secured cluster services for RHACS Cloud Service on other platforms
+include::modules/common-attributes.adoc[]
+:context: install-secured-cluster-cloud-other
+:topic-helm:
+
+toc::[]
+
+[role="_abstract"]
+You can install {product-title-managed-short} on your secured clusters by using one of the following methods:
+
+* Install by using Helm charts
+* Install by using the `roxctl` CLI (do not use this method unless you have a specific installation need that requires using it)
+
+:FeatureName: {product-title-managed-short}
+include::snippets/field-trial.adoc[leveloffset=+1]
+
+[id="installing-sc-helm-cloud-other"]
+== Installing {product-title-managed-short} on secured clusters by using Helm charts
+
+You can install {product-title-short} on secured clusters by using Helm charts with no customization, by using Helm charts with the default values, or by using Helm charts with customizations of configuration parameters.
+
+##Please confirm that we need the customizations topics - will users need this? Or will they install with no customizations##
+
+First, ensure that you add the Helm chart repository.
+
+include::modules/adding-helm-repository.adoc[leveloffset=+2]
+
+[id="installing-sc-helm-default-cloud-other"]
+=== Installing {product-title-managed-short} on secured clusters by using Helm charts without customizations
+
+include::modules/acs-quick-install-secured-cluster-using-helm.adoc[leveloffset=+3]
+
+[id="configure-secured-cluster-services-helm-chart-customizations-cloud-other"]
+=== Configuring the secured-cluster-services Helm chart with customizations
+
+This section describes Helm chart configuration parameters that you can use with the `helm install` and `helm upgrade` commands.
+You can specify these parameters by using the `--set` option or by creating YAML configuration files.
+
+Create the following files for configuring the Helm chart for installing {product-title}:
+
+* Public configuration file `values-public.yaml`: Use this file to save all non-sensitive configuration options.
+* Private configuration file `values-private.yaml`: Use this file to save all sensitive configuration options. Make sure that you store this file securely.
+//TODO: Add links to sample configuration files.
```

Comment:
Are we still doing the TODO?

Reply:
I think that's left over from before my time, so probably not :)